backend/Match/models.py

```diff
@@ -19,7 +12,8 @@ class Match(models.Model):
 	created_at = models.DateTimeField(auto_now_add=True)
 	updated_at = models.DateTimeField(auto_now=True)
 
-	objects = MatchManager()
+	tournament = models.ForeignKey(Tournament, related_name='matches', null=True, blank=True, default=None, on_delete=models.CASCADE)
+	tournament_match_id = models.PositiveIntegerField(default=0)
```

Comment:
Why two keys? Just one foreing key should be enoudh, and the cascading will probably cause na issue again with stats etc

Reply:
Ah I see... the column could be next_match etc, so just a foreing key to the next match.

Reply:
but how can it be a foreign key to next_match? we need to know in what order these matches are in the tournament.

Reply:
The same way as any other foreign key is the whole table?

Reply:
Like the same way we have foreign key to the tournament table we can have a foreign key to another row in the Match table.